ros/lib/utils.py

```diff
@@ -31,6 +31,16 @@ def get_or_create(session, model, keys, **kwargs):
     return instance
 
 
+def delete_record(session, model, **system_fields):
+    keys = list(system_fields.keys())
+    instance = session.query(model).filter_by(**{k: system_fields[k] for k in keys}).first()
+    inventory_id = instance.inventory_id
+    if instance:
+        session.delete(instance)
+        session.commit()
+    return inventory_id
```

Comment:
Lets make `delete_record` function more generic and not specific to systems table. 

```suggestion
def delete_record(session, model, **kwargs):
    keys = list(kwargs.keys())
    instance = session.query(model).filter_by(**{k: kwargs[k] for k in keys}).first()
    if instance:
        session.delete(instance)
        session.commit()
    return instance
```